tests/unit/amo/components/TestSearchResult.js

```diff
@@ -131,19 +131,170 @@ describe(__filename, () => {
   });
 
   it('adds a theme-specific class', () => {
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      type: ADDON_TYPE_THEME,
-      theme_data: {
-        previewURL:
-          'https://addons.cdn.mozilla.net/user-media/addons/334902/preview_large.jpg?1313374873',
-      },
+    const root = render({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        theme_data: {
+          previewURL:
+            'https://addons.cdn.mozilla.net/user-media/addons/334902/preview_large.jpg?1313374873',
+        },
+      }),
     });
-    const root = render({ addon });
 
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('does not render an image if the isAllowedOrigin is false', () => {
+    const differentOriginUrl = 'http://example.com';
+    const isAllowedOriginSpy = sinon.spy();
```

Comment:
This is the right idea, but we don't actually need to verify that the `isAllowedOriginSpy` was called, rather, we need to make sure that the `isAllowedOriginSpy` returns `false`, so you could use a `sinon.stub` and make sure that it returns `false`, like so:

`const isAllowedOriginSpy = sinon.stub().returns(false);`

Reply:
I *think* I got it. let me know if the update is okay : )